src/mkdocstrings/plugin.py

```diff
@@ -289,7 +289,7 @@ def _load_inventory(cls, loader: Callable[..., Iterable[Tuple[str, str]]], url:
             A mapping from identifier to absolute URL.
         """
         log.debug(f"Downloading inventory from {url!r}")
-        req = urllib.request.Request(url, headers={"Accept-Encoding": "gzip"})
+        req = urllib.request.Request(url, headers={"Accept-Encoding": "gzip", "User-Agent": "mkdocstrings/0.15.0"})
```

Comment:
No, actually I found that most Python API sites reject access without a user agent.
How to update this version number consistently is another organizational topic that I don't have the answer to right now.

Reply:
Oh OK. Then maybe we should depend on importlib-metadata so we can retrieve the current version of mkdocstrings.

Reply:
I'd prefer a `__version__` constant somewhere (which we'd still have to maintain manually or in some other way). I find importlib-metadata handling to be fragile. Because... at the moment mkdocstrings can be used without installing it  (and I often use modules as such) but with that it has a dependency on being installed.
In any case, I think it'd be best handled separately, and in this PR to be kept as is.

Reply:
Not a fan of `__version__`. Lets discuss it in another issue/PR indeed :)